src/helpers/assetHelper.ts

```diff
@@ -17,59 +30,69 @@ export const getCoverPath = ({
   path = '/imgs/default-cover.jpg',
   format,
   device,
-  pixelRatio = 1,
+  pixelRatio,
+  extension = DEFAULT_EXTENSION_FOR_IMAGES,
+  position = IMAGE_POSITIONS.CENTER,
 }: {
   path?: string;
-  format: ImageFormatEnum;
-  device: DeviceEnum;
-  pixelRatio?: number;
+  format: ImageFormatType;
+  device: DeviceType;
+  pixelRatio: number;
+  extension?: ImageExtensionType;
+  position?: ImagePositionType;
 }): string => {
   const isProd: boolean = process.env.NODE_ENV === 'production';
   const directoryPath = dirname(path);
   const filename = basename(path, extname(path));
-  const imageFormat = IMAGE_FORMATS[device][format];
+  const imageFormat = SIZES_BY_IMAGE_FORMAT[device][format];
 
   const pathFile = isProd
-    ? `${directoryPath}/${filename}-w${imageFormat.width * pixelRatio}-h${imageFormat.height * pixelRatio}.avif`
-    : `${path}?width=${imageFormat.width * pixelRatio}&height=${imageFormat.height * pixelRatio}&format=avif`;
+    ? `${directoryPath}/${filename}-w${imageFormat.width}-h${imageFormat.height}-x${pixelRatio}.${extension}`
+    : `${path}?width=${imageFormat.width}&height=${imageFormat.height}&pixelRatio=${pixelRatio}&position=${position}&format=${extension}`;
```

Comment:
why is the path different depending on the env? I would keep only one image generation process for all envs, as it is in prod

Reply:
Car en local, on ne souhaite pas perdre de temps à générer les images de plus en local on est full SSR et non en static